app/views/shared/_table.html.erb

```diff
@@ -21,9 +21,10 @@
 </table>
 
 <script>
-window.onload = function() {
-  $('#data-table').DataTable({
-    "order": [[ 1, "desc" ]]
-  });
-};
+ window.onload = function() {
+   $('#data-table').DataTable({
+     "order": [[ 1, "desc" ]],
+     "bStateSave": true
```

Comment:
Are you sure `bStateSave` exists? I couldn't find it documented.

Also, I could be reading this diff wrong, but it looks like you indented this javascript by 1 space.

Reply:
I think it is just `stateSave` now:
https://datatables.net/reference/option/stateSave

Reply:
I'll check it out and update.  Thanks!